shared/src/commonMain/kotlin/com/bunbeauty/shared/domain/feature/addition/GetIsAdditionsAreEqualUseCase.kt

```diff
@@ -0,0 +1,22 @@
+package com.bunbeauty.shared.domain.feature.addition
+
+import com.bunbeauty.shared.domain.model.cart.CartProduct
+
+class GetIsAdditionsAreEqualUseCase {
+    operator fun invoke(
+        initialCartProduct: CartProduct?,
+        additionUuidList: List<String>,
+    ): Boolean {
+        // Check if the lists have the same size
+        val listsHaveSameSize =
+            initialCartProduct?.cartProductAdditionList?.size == additionUuidList.size
+
+        // Check if each element in 'additionList' has a corresponding element in 'cartProductAdditionList' with the same 'uuid'
```

Comment:
> `list1` has a corresponding element in `list2`

This doesn't mean that `list1` has the same elements as `list2`. 
First of all, you can just use `list1.containsAll(list2)`. But that is not what we need. 
For example, 
```
// Input:
initialCartProduct = [1, 2, 3]
additionUuidList = [1, 2]
// Output:
true
```

Reply:
Let's write something like `list1.size == list2.size && list1.sorted() == list2.sorted()`

Reply:
Let's also add this case ☝️ to tests

Reply:
✅ fixed